docs/source/copyright.rst

```diff
@@ -18,3 +19,4 @@ The current main developers of Eureka! are (ordered alphabetically by first name
 - Taylor Bell
 - Yoni Brande
 
+We would also like to acknowledge the valuable contributtions from numerous users that have helped make Eureka! a success.  Thank you!
```

Comment:
What do you think about:
```
We would also like to acknowledge the valuable contributions from the numerous users that have helped make Eureka! a success, the full list of which can be found `here<https://github.com/kevin218/Eureka/graphs/contributors>`_. Thank you!
```

Also not the typo in contributions (contributtions).